firebase/firebase-cpp-sdk
# Firestore integration tests on tvos

Changes done to XCode project for firestore that allows us to run integration tests on device and simulator.

added a tvos target to XCode project for storage.
updated Podfile to build pods for tvos platform for this newly added target
To test this,

Make sure to have a valid GoogleService-Info.plist.
pod install
open integration_test.xcworkspace
Drag and drop firebase.xcframework, firebase_auth.xcframework and firebase_firestore.xcframework

All tests pass on simulator and device. Exposed a problem in packaging with missing some required firestore headers.
